internal/pkg/template/input/step.go

```diff
@@ -0,0 +1,18 @@
+package input
+
+type StepsGroups []StepsGroup
+
+type StepsGroup struct {
+	Description string `json:"description" validate:"max=80"`
+	Required    string `json:"required" validate:"oneof=all atLeastOne exactOne zeroOrOne optional"`
+	Steps       []Step `json:"steps" validate:"min=1,dive"`
+}
+
+type Step struct {
+	Icon              string `json:"icon"`
+	Name              string `json:"name" validate:"required,max=20"`
+	Description       string `json:"description" validate:"max=40"`
+	DialogName        string `json:"dialogName,omitempty" validate:"max=20"`
```

Comment:
Do validate by malo tiez ist `omitempty`, inak to spadne.
Podobne aj pri `DialogDescription`.